build-system/compile/compile.js

```diff
@@ -236,15 +236,32 @@ function compile(
     }
     externs.push('build-system/externs/amp.multipass.extern.js');
 
+    // Normally setting this server-side experiment flag would be handled by
+    // the release process automatically. Since this experiment is actually on the
+    // build system instead of runtime, we never run it through babel and therefore
+    // must compute it here.
+    const isStrict =
+      argv.define_experiment_constant &&
```

Comment:
There's no need to test for this explicitly, the second check will work on its own.

Reply:
i have no idea why I wrote that